associated-token-account/program-test/Cargo.toml

```diff
@@ -16,4 +16,4 @@ solana-program-test = "1.16.1"
 solana-sdk = "1.16.1"
 spl-associated-token-account = { version = "1.1", path = "../program", features = ["no-entrypoint"] }
 spl-token = { version = "4.0", path = "../../token/program", features = ["no-entrypoint"] }
-spl-token-2022 = { version = "0.6", path = "../../token/program-2022", features = ["no-entrypoint"] }
+spl-token-2022 = { version = "1.0", path = "../../token/program-2022", features = ["no-entrypoint"] }
```

Comment:
before 1.0, minor versions function like major versions, so we can bump this 0.7 -- we'll go to 1.0 for the first official recommended mainnet release

Reply:
Copy, going with 0.7